etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy

```diff
@@ -66,23 +71,17 @@ class EncounterMigrator extends SqlMigrator {
             UPDATE hivmigration_encounters SET encounter_uuid = uuid();
         ''')
 
-        executeMysql("Fill encounter types column", '''
-            SET @encounter_type_intake = (select encounter_type_id from encounter_type where uuid = 'c31d306a-40c4-11e7-a919-92ebcb67fe33');
-            SET @encounter_type_followup = (select encounter_type_id from encounter_type where uuid = 'c31d3312-40c4-11e7-a919-92ebcb67fe33');
-            SET @encounter_type_lab_results = (select encounter_type_id from encounter_type where uuid = '4d77916a-0620-11e5-a6c0-1697f925ec7b');
-            SET @encounter_type_drug_dispensing = (select encounter_type_id from encounter_type where uuid = 'cc1720c9-3e4c-4fa8-a7ec-40eeaad1958c');
-            SET @encounter_type_drug_order = (select encounter_type_id from encounter_type where uuid = '0b242b71-5b60-11eb-8f5a-0242ac110002');
-            
+        executeMysql("Fill encounter types column", '''            
             UPDATE hivmigration_encounters SET encounter_type_id = CASE
-                WHEN source_encounter_type = "intake" THEN @encounter_type_intake
-                WHEN source_encounter_type = "followup" THEN @encounter_type_followup
-                WHEN source_encounter_type = "lab_result" THEN @encounter_type_lab_results
-                WHEN source_encounter_type = "anlap_lab_result" THEN @encounter_type_lab_results
-                WHEN source_encounter_type = "accompagnateur" THEN @encounter_type_drug_dispensing
-                WHEN source_encounter_type = "regime" THEN @encounter_type_drug_order
+                WHEN source_encounter_type = "intake" THEN encounter_type('HIV Intake')
```

Comment:
Where is this encounter_type function defined?  Does this come from something we use for reporting?

Reply:
I think it must. @dmdesimone ?